rhizome/lib/vm_setup.rb

```diff
@@ -221,6 +222,47 @@ def boot_disk(boot_image)
     FileUtils.chown @vm_name, @vm_name, vp.boot_raw
   end
 
+  def spdk
+    r "mkdir #{vp.q_hugepages}"
+    r "mount -t hugetlbfs -o uid=#{q_vm},size=1G nodev #{vp.q_hugepages}"
+    r "chown -R #{q_vm} #{vp.q_hugepages}"
+
+    vp.write_spdk_config(<<EOS)
```

Comment:
Should we get away from string templating and instead use a data structure and JSON generation? With YAML I know I did this, because I wanted to refer to YAML literals in the manual, but with JSON it's so close to the ruby literal that it has less of a notation-swapping penalty.

I also wanted to get rid of our YAML templating once it seemed like it hasn't changed in a while and we didn't anticipate it changing soon. We may soon be there, but fresh work on IPv4 and virtual networking may be one last unstable moment.

Reply:
done